modules/checking-mco-status-certs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="checking-mco-status-certs_{context}"]
+= Viewing and interacting with certificates
+
+The following certificates are handled in the cluster by the Machine Config Controller (MCC) and can be found in the ControllerConfig:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
+Use the following procedure to get information about the certificates, including the underyling bundle the certificate comes from, and the signing and subject data.
+
+.Procedure
+
+* Run the following command to get certificate information:
++
+[source,terminal]
+----
+$ oc describe controllerconfig/machine-config-controller
+----
++
+.Expected output
++
+[source,text]
+----
+"controllerCertificates": [
+                   {
+                       "bundleFile": "KubeAPIServerServingCAData",
+                       "signer": "<signer_data1>",
+                       "subject": "CN=openshift-kube-apiserver-operator_node-system-admin-signer@168909215"
+                   },
+                   {
+                       "bundleFile": "RootCAData",
+                       "signer": "<signer_data2>",
+                       "subject": "CN=root-ca,OU=openshift"
+                   }
+                ]
+----
+
+* Alternatively, you can access a simpler version of the information found in the ControllerConfig by checking the machine config pool status:
```

Comment:
Since this is presented as an either or, I don't think you need alternatively here. You can show what the either/or options are by wording the starting part of the sentence. And make them more parallel to each other. Depending on which way you chose for the two options I gave for the first bullet, could be something like:

```suggestion
* To get a simple version of the certificate information, run the following command:
```